src/cli/README.md

```diff
@@ -1164,6 +1202,23 @@ Done
  - RSSI: -18 (dBm) (Exponential Moving Average)
 ```
 
+### linkmetrics query \<ipaddr\> forward \<seriesid\>
+
+Perform a Link Metrics query (Forward Tracking Series).
+
+- ipaddr: Peer address.
+- seriesid: The Series ID.
+
+```bash
+> linkmetrics query fe80:0:0:0:3092:f334:1455:1ad2 forward 1
+Done
+> Received Link Metrics Report from: fe80:0:0:0:3092:f334:1455:1ad2
+
+ - LQI: 76 (Exponential Moving Average)
+ - Margin: 82 (dB) (Exponential Moving Average)
+ - RSSI: -18 (dBm) (Exponential Moving Average)
```

Comment:
If it is the report for the forward tracking series `1` configured above
`linkmetrics mgmt fe80:0:0:0:3092:f334:1455:1ad2 forward 1 dra pqmr`

why there is no sub report for `p`?

Reply:
The result isn't actually related with the previous example. But I can add the sub report for `p`.